README.md

```diff
@@ -68,6 +68,10 @@ Currently the most reliable way of installing Gatekeeper is to build and install
    * make sure your kubectl context is set to the desired installation cluster
    * run `make deploy`
 
+#### Deploying via Helm ####
+
+A basic Helm template exists in `chart/gatekeeper-operator`. If you have Helm installed and Tiller initialized on your cluster you can deploy via `helm install chart/gatekeeper-operator/`
```

Comment:
Lets decide on if we want to use helm v2 vs helm v3 as the instructions and experience are different.

Reply:
Can we stick with v2 as that is currently the latest stable version (and what this all works with)? I can note this in the README.

Reply:
helm v3 is released, so it would be best to support both